source/Octopus.Server.Client/Model/Endpoints/TentacleEndpointConfigurationResource.cs

```diff
@@ -27,9 +27,6 @@ protected TentacleEndpointConfigurationResource(string thumbprint, string uri)
         [Writeable]
         public string Uri { get; set; }
 
-        [Writeable]
-        public TentacleDetailsResource TentacleVersionDetails { get; set; }
```

Comment:
I've removed this as it's no longer required for Kubernetes Tentacles. The information stored in this property is now stored in a combination of `KubernetesAgentDetails` and `UpgradeInformation` properties directly on the `KubernetesTentacleEndpointResource`.